### ISSUE TITLE
Print message after outputing markdown report

### ISSUE THREAD

**Issue by felipesanches** [googlefonts/fontbakery#2050]
Some users at the AtypI 2018 workshop had the impression that the markdown file was not created with their report because nothing regarding that show up within the text terminal messages.

We should explicitly say something like: "The report file was written to ~/Desktop/Something.md"

**Comment by davelab6** [comment]
This is not "this unix way" but I agree that we should do that. 

Its even possible that the markdown output can become a default, if it can be set up in a way that it looks fine when read as plain text...

**Comment by davelab6** [comment]
Agree, I think the markdown arg and the file name arg should be separated
and composable. Possibly even a "tee" option to do both

**Comment by felipesanches** [comment]
@n7s told us at https://github.com/googlefonts/fontbakery/issues/2050#issuecomment-421586519 that some users may get confused about the --ghmarkdown option expecting the markdown report to be output on stdout and then trying to use an output redirection with `>`.

But doing that causes an argparse error and results in displaying the usage instructions, so I think it is fine the way it is nowadays.

![screenshot at 2018-09-28 17 28 32](https://user-images.githubusercontent.com/213676/46231961-2da59000-c35d-11e8-8fc3-40cad3ba2e4b.png)

**Comment by felipesanches** [comment]
I have just added the messages informing the file path where the reports were saved. Both for JSON and for Markdown output.

![screenshot at 2018-09-28 17 31 14](https://user-images.githubusercontent.com/213676/46232061-76f5df80-c35d-11e8-902f-a2eb5a4a6a94.png)

**Comment by davelab6** [comment]
> I think the markdown arg and the file name arg should be separated and composable. Possibly even a "tee" option to do both

I think its very reasonable to expect that the markdown output arg will effect the output style, from plain text to markdown, but in both cases the output will be to stdout by default (and thus can be redirected to a file using `>`), and there should be another arg for writing output to a file for people who want to set the filename by arg instead of using shell piping for whatever reason.

**Comment by felipesanches** [comment]
If we did it, then the user would get no text terminal output for a couple minutes while the whole set of checks are run and then would suddenly get all the markdown output at once. That would be terrible in terms of usability. It seems to me that outputing markdown as the checks are being executed would require too much effort to redesign the check-runner to achieve that and so I think it is not worth doing so.

The proper usage is well documented and the user will get the usage message explaining how to pass the correct arguments in all conceivable scenarios of a user incorrectly invoking fontbakery's `--ghmarkdown` option.

**Comment by felipesanches** [comment]
@davelab6 Please close the issue if you agree.

**Comment by davelab6** [comment]
> That would be terrible in terms of usability

@n7s what do you think?

I think that it is fine in terms of usability, because if someone wants to see the colorful output with animating progress bar, they can use the regular terminal mode; if they are asking for markdown, they can be assumed to understand how unix command prompt interaction typically works. 

The problem @n7s encountered is indeed that he understands how unix command prompt interaction typically works, and was thrown off because FB currently does not act like a normal unix program. 

However, if you want to be sure that users get a positive confirmation that their command is running, you can output a note to STDERR instead of STDOUT so that when a user pipes the output using `>` it will not be written to their file. 

Eg,

    $ ./fontbakery family/* --ghmarkdown > report.md
    Font Bakery is running, this will take a few minutes...
    Font Bakery run complete!
    $ 

> the user will get the usage message explaining how to pass the correct arguments in all conceivable scenarios of a user incorrectly invoking fontbakery's `--ghmarkdown` option.

I don't agree that is the case, if the message is

    fontbakery check-googlefonts: error: argument --ghmarkdown: expected one argument

I think that would be the case of the message was

    fontbakery check-googlefonts: error: argument --ghmarkdown: expected one argument, a filename to write the markdown report into. try again with, 
    
    ./fontbakery path/used/in/last/command --args-used-in-last-command --ghmarkdown report.md

**Comment by davelab6** [comment]
Anyway this is not very important; you said that the resolution was

> We should explicitly say something like: "The report file was written to ~/Desktop/Something.md"

and that's a quick thing, and its now done, so I'm retagging this as a P4 and moving to the MISC milestone :)
